.gitignore

```diff
@@ -1,4 +1,3 @@
-Manifest.toml
```

Comment:
I warmly recommend keeping the Manifest.toml ignored. You can always force adding ignored files to git with `git add -f`, if that's why you removed the ignore line.

You can also ignore only the Manifest.toml in the top directory with
```
/Manifest.toml
```
which could be a good solution

Reply:
Right, I'll probably just ignore the top level manifest file.